themes/bootstrap3/templates/RecordDriver/DefaultRecord/link-collection.phtml

```diff
@@ -1 +1,5 @@
-<?=$this->url('collection', ['id' => $this->lookfor])?>?recordID=<?=urlencode($this->driver->getUniqueID())?>
\ No newline at end of file
+<?=$this->url(
+    'collection',
+    ['id' => $this->lookfor],
+    ['query' => ['recordID' => $this->driver->getUniqueID(), 'sid' => ($this->results ? $this->results->getSearchId() : null) ?? $this->searchMemory()->getCurrentSearchId()]]
```

Comment:
I've seen this pattern of `($this->results ? $this->results->getSearchId() : null) ?? $this->searchMemory()->getCurrentSearchId()` in more than once place, if memory serves... It's not the most readable thing. Would it make any sense to either add an optional $results parameter to getCurrentSearchId(), or add a separate getCurrentSearchIdFromResultsOrMemory method to make this a little cleaner? (Feel free to ignore if you don't think it's a worthwhile change).

Reply:
I think it's only in one other place and there the alternative is getLastSearchId, so the cases don't quite match. I'd revisit this if we end up needing it in more places.